src/index.ts

```diff
@@ -20,6 +38,45 @@ export = (app: Application) => {
       await context.github.issues.createComment(prComment)
     }
   })
+
+  const checkForInvalidVersions = (entries: PullRequestsGetFilesResponseItem[]): boolean => {
+    app.log('getting files')
+    let firstVersion: PartialSwagger | undefined
+    let isInvalid: boolean = false
+
+    entries.forEach(element => {
+      app.log(`file found for path: ${element.contents_url}`)
+      const partialSwagger = swaggerPathScraper.parsePathToSwagger(element.contents_url)
+      if (partialSwagger !== undefined) {
+        if (firstVersion !== undefined) {
+          if (
+            partialSwagger.rpName !== firstVersion.rpName ||
+            partialSwagger.version !== firstVersion.version
+          ) {
+            // more than one RP or version was detected.  Jump out of loop and return true
+            app.log(`multiple versions or RPs detected with ${element.contents_url}`)
+            isInvalid = true
```

Comment:
should we return in this case? as the first invalid we find, makes it unsupported.

Reply:
I put a return in there and the TypeScript complained that not all paths returned a value.  It seemed like a strange error but didn't pick up on the fact that the overall function returns a value.